nano/ipc_flatbuffers_lib/flatbuffers-build/flatbuffers.pc

```diff
@@ -3,7 +3,7 @@ includedir=/usr/local/include
 
 Name: FlatBuffers
 Description: Memory Efficient Serialization Library
-Version: 1.12.0
+Version: 23.3.3
```

Comment:
Why was the flatbuffers dependency updated? If this is necessary, let's put it into a new PR